<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDisplayOptionWidget</class>
 <widget class="QWidget" name="ChatDisplayOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatDisplayOptionWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QCheckBox,QComboBox
{
	font-family:&quot;微软雅黑&quot;;
	color:#666666;
}
QPushButton
{
	border:none;
}

QPushButton:disabled
{
	background:#666666;
}

QComboBox {
    border: 1px solid #EBEBEB;
    border-radius: 0px;
    padding: 1px 18px 1px 3px;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: #AAFFFFFF;
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
	border-left-width: 1px;
    border-left-color: #EBEBEB;
    border-left-style: solid; /* just a single line */
}

QComboBox::down-arrow {
    image: url(:/Images/jietu_normal.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="dialogbg" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TitlebarWidget" name="titlebar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>60</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>文字设置</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>140</y>
              <width>60</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>气泡设置</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>20</y>
              <width>40</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>字体</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>50</y>
              <width>40</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>字号</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>79</x>
              <y>80</y>
              <width>61</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>我的字体</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>79</x>
              <y>150</y>
              <width>61</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>我的颜色</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>79</x>
              <y>180</y>
              <width>61</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>对方颜色</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QFontComboBox" name="cbFontFamily">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>20</y>
              <width>200</width>
              <height>22</height>
             </rect>
            </property>
            <property name="currentFont">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
           </widget>
           <widget class="QComboBox" name="cbFontSize">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>50</y>
              <width>200</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
           </widget>
           <widget class="QPushButton" name="btnFontColor">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>80</y>
              <width>200</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btnMyColor">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>150</y>
              <width>201</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btnOtherColor">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>180</y>
              <width>201</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>79</x>
              <y>110</y>
              <width>61</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>对方字体</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="btnOtherFontColor">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>110</y>
              <width>200</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
